service/create/service.go

```diff
@@ -1686,3 +1401,16 @@ func (s *Service) updateFunc(oldObj, newObj interface{}) {
 
 	s.logger.Log("info", fmt.Sprintf("scaling workers auto scaling group from %d to %d", oldSize, newSize))
 }
+
+func (s *Service) uploadCloudconfigToS3(svc *s3.S3, s3Bucket, path, data string) error {
+	if _, err := svc.PutObject(&s3.PutObjectInput{
+		Body:          strings.NewReader(data),
+		Bucket:        aws.String(s3Bucket),
+		Key:           aws.String(path),
+		ContentLength: aws.Int64(int64(len(data))),
+	}); err != nil {
+		return microerror.Mask(err)
+	}
+
+	return nil
+}
```

Comment:
This method is moved after the func handlers. Again as prep to make the migration easier.